metricbeat/module/system/memory/memory.go

```diff
@@ -3,46 +3,42 @@
 package memory
 
 import (
-	"github.com/elastic/beats/metricbeat/helper"
-
 	"github.com/elastic/beats/libbeat/common"
-	"github.com/elastic/beats/libbeat/logp"
+	"github.com/elastic/beats/metricbeat/mb"
 	"github.com/elastic/beats/topbeat/system"
+	"github.com/pkg/errors"
```

Comment:
I pretty much destroyed any import ordering conventions you may have had. I'm happy to fix all imports to conform to a single convention. I just need the rules to the conventions.

I believe your conventions advise three import blocks: standard lib imports, "github.com/elastic/beats" imports, and then everything else?